Core/GDCore/Events/Tools/EventsCodeNameMangler.cpp

```diff
@@ -57,11 +65,30 @@ const gd::String& EventsCodeNameMangler::GetExternalEventsFunctionMangledName(
       partiallyMangledName.replace(i, 1, "_" + gd::String::From(unallowedChar));
     }
   }
+  return partiallyMangledName;
+}
 
-  mangledExternalEventsNames[externalEventsName] = "GDExternalEvents" + partiallyMangledName;
-  return mangledExternalEventsNames[externalEventsName];
+gd::String EventsCodeNameMangler::GetMangledName(

```

Comment:
This method doesn't handle non-latin character as well as the one that was used in the JS code, but the IDE forbid these characters anyway.
It should probably be factorized with what the code generator use on function names.